gulpfile.js

```diff
@@ -57,11 +61,41 @@ gulp.task('default', function () {
   console.log('    The name of the package to publish. If no package is specified, then all packages will be published.');
 });
 
+gulp.task("install", function () {
+  if (!package) {
+    console.log(`No --package specified to run "npm install" on.`);
+  } else {
+    const nodejsReadmeFilePaths = findReadmeNodejsMdFilePaths(azureRestAPISpecsRoot);
+
+    let foundPackage = false;
+
+    for (let i = 0; i < nodejsReadmeFilePaths.length; ++i) {
+      const nodejsReadmeFilePath = nodejsReadmeFilePaths[i];
+
+      const nodejsReadmeFileContents = fs.readFileSync(nodejsReadmeFilePath, 'utf8');
+      const packageName = getPackageNameFromReadmeNodejsMdFileContents(nodejsReadmeFileContents);
+
+      if (package === packageName || packageName.endsWith(`-${package}`)) {
```

Comment:
What's the case when second condition is true?

Reply:
Our published packages look like `azure-arm-storage`, so the first condition is if I entered `gulp install --package azure-arm-storage` and the second condition is if I entered `gulp install --package storage`